api/feedback/v1/feedback.proto

```diff
@@ -31,8 +31,7 @@ message GetSurveysRequest {
     min_items : 1,
     items : {enum : {defined_only : true, not_in : 0}}
   } ];
-  // future: have rules on whether a user should see the feedback survey
-  string user = 2;
```

Comment:
some proto cleanup. decided to remove the fields we aren't going to use in the V1